src/styles/main.scss

```diff
@@ -1,3 +1,67 @@
 body {
   margin: 0;
 }
+
+$size: 100px;
+$shift: 10px;
+$gap-shift: 1px;
+$shift-border: 1px;
+$number-size: 0.3 * $size;
+$base-shift: (
+  $size - $number-size) /2;
+$day-name: (
+  "mon": 1,
+  "tue": 2,
+  "wed": 3,
+  "thu": 4,
+  "fri": 5,
+  "sat": 6,
+  "sun": 7
+);
+
+@function getWidth ($size, $shift, $small-shift) {
+  @return ($size * 7)+($shift + $gap-shift)+($shift-border*14);
+}
```

Comment:
create separate files for scss features(variables, functions and so on), BEM blocks, reset